src/Uno.Extensions.Maui.UI/MauiHost.cs

```diff
@@ -80,7 +82,11 @@ private static void OnSourceChanged(DependencyObject dependencyObject, Dependenc
 	private static ILogger GetLogger() =>
 		IPlatformApplication.Current?.Services.GetRequiredService<ILogger<MauiHost>>() ?? throw new MauiEmbeddingInitializationException();
 
-	private VisualElement? EmbeddedView;
+	private object NativeView
```

Comment:
Having this property as a wrapper around base.Content is only necessary because we're overloading Content. If we change the name of the property to `MauiContent` we can remove this wrapper property too.